src/Plivo/Client/HttpClient.cs

```diff
@@ -103,6 +103,11 @@ public string GetAuthId()
         public void SetTimeout(int timeout)
         {
             ((SystemHttpClient) _client)._client.Timeout = TimeSpan.FromSeconds(timeout);
+            ((SystemHttpClient) _client)._voiceBaseUriClient.Timeout = TimeSpan.FromSeconds(timeout);
+            ((SystemHttpClient) _client)._voiceFallback1Client.Timeout = TimeSpan.FromSeconds(timeout);
+            ((SystemHttpClient) _client)._voiceFallback2Client.Timeout = TimeSpan.FromSeconds(timeout);
+            ((SystemHttpClient) _client)._callInsightsclient.Timeout = TimeSpan.FromSeconds(timeout);
```

Comment:
@SaiTharunPlivo, Why are we doing this for callinsights client and lookup client ? As discussed, we are supposed to do this for only voicebaseuri and voicefallback clients.